Back at it Again

Exo-212 opened the pull request:
Woo! Going back into Pubby for another round of dopplerfying.

Here's the highlights:
-Armory: Civilian-Issue lockers have been moved to EVA Storage. An empty Grenade Launcher, an Empty grenade Belt, and three empty bandoliers have been added.
-EVA Storage: The airlock now requires no access on Red Alert. Contents have been changed to help crew assist with threats. The Civilian Issue Gear (six sets of Frontier Armor, Frontier Helmet, and Mini-Egun) are in security-access lockers, requiring a secoff or a command member with appropriate access to open. There is now two biohazard lockers and two radiation lockers.
-Medical: Switched the location of Paramedic Dispatch and Security office. Shrank New Security office for more Treatment Center room. Treatment center now has a crew-facing airlock that requires no access on Red Alert. A set of shutters are available on medical access if this needs to be locked.
-Chemistry: Returned the chem factory room to maintenance. R.I.P. Pharmacy now has a storage room with some bulk chemical bags from reliable sources.
-Library: Adds three docking ports(!) to the library to bring life to this desolate, remote part of the station. Shuttle owners rejoice, you might be able to find parking.
-Telecoms: Moved the whole Telecoms sat further away from the station. Connected the sat to the distro network. Vent-crawlers rejoice! You can make it to Communications.
-Monastery: Also Connected to distro.
-Cargo: Added a "Sikth & Kurabi Stationery Supplies ™ Uplink" Console to the lobby, right in the view of the security checkpoint.  It's legal! All the best lawyers say so! Also moved the frontier fabricator next to the protolathe.
-Entertainment Monitors: Added monitors in/around break rooms and public areas. Remember that you can mute them if you get annoyed.
-Shared Gas Storage: Replaced 3 Oxygen Canisters with 3 Water Vapour Canisters
-Atmospherics: Placed gas miners of same type in each gas tank. (O2,N2,C02,N20,Plasma)

OmegaDarkPotato commented:
Massive thumbs down for the swap of the sec / paramedic dispatch office locations, paramedic dispatch looking over the lobby meant that doctors could sit in there and use it as a reception desk. We no longer have a reception in the lobby and this suuuucks. I need my reception desk to live

Exo-212 commented:
> I noticed that a lot of the cabinets you placed in command offices are the loot spawner cabiners and not the, actual cabinets. That was an earlier mistake though, I just never bothered to report it

This is an intentional choice.


> Massive thumbs down for the swap of the sec / paramedic dispatch office locations in med, paramedic dispatch looking over the lobby meant that doctors could sit in there and use it as a reception desk. We no longer have a reception in the lobby and this suuuucks. I need my reception desk to live

I hear you. This shuffle was inspired by seeing medical guards sit around and see little interaction while at their post.

I'm still committed to the location change, but I've moved the Health Kiosk and Medical Supplies deeper into the lobby and removed the double-airlock. This is intended to make entering the inner-lobby - and within sight of paramedic dispatch - more appealing.
